client/components/swipeable/style.scss

```
@@ -0,0 +1,32 @@
+@import '@wordpress/base-styles/mixins';
+.swipeable__container {
+	width: 100%;
+	height: 100%;
+	margin-left: auto;
+	margin-right: auto;
+	position: relative;
+	overflow: hidden;
+	list-style: none;
+	padding: 0;
+	z-index: 1;
+	touch-action: pan-y;
+}
+
+.swipeable__pages {
+	position: relative;
+	width: 100%;
+	height: 100%;
+	z-index: 1;
+	display: flex;
+
+	transition: all 0.5s ease-in-out;
+	transition-property: transform, height;
+	@include reduce-motion( 'transition' );
+}
+.swipeable__resize-observer {} /* stylelint-disable-line block-no-empty */
+
+.swipeable__page {
+	width: 100%;
+	flex-shrink: 0;
+	margin-left: 0;
+}
```

Nit: adding a trailing empty line
```suggestion
}

```